pkg/datastore/commandstore.go

```diff
@@ -120,3 +110,19 @@ func (s *commandStore) UpdateCommand(ctx context.Context, id string, updater fun
 		return p.Validate()
 	})
 }
+
+func (s *commandStore) MarkAsTimeOut(ctx context.Context, id string) error {
+	return s.update(ctx, id, func(c *model.Command) error {
+		c.Status = model.CommandStatus_COMMAND_TIMEOUT
+		return nil
+	})
```

Comment:
Add a private `update` function to ensure that `UpdatedAt` field is being updated.

Reply:
yeh, it's 👍